hedgehog/src/Hedgehog/Gen.hs

```diff
@@ -768,6 +769,16 @@ bytes range =
 ------------------------------------------------------------------------
 -- Combinators - Choice
 
+-- | Trivial generator that always produces the same element.
+--
+-- All of the following expressions are equivalent:
+--
+-- * @'constant' a@
+-- * @'pure' a@
+-- * @'element' [a]@
```

Comment:
For `discard` which serves a similar purpose, I have used the words `This is another name for 'empty' / 'mzero'.`

So just for consistency I'd prefer this to say:
```
-- | Trivial generator that always produces the same element.
--
--   /This is another name for 'pure' \/ 'return'./
```